prep-javascript-functions/main.js

```diff
@@ -0,0 +1,62 @@
+function addTwoNumbers(x, y) {
+  return x + y;
+}
+
+var sum = addTwoNumbers(4, 4);
+console.log('sum', sum);
+
+function converHoursToMinutes(x, y) {
+  return x * y;
+}
+
+var minutes = converHoursToMinutes(2, 60);
+console.log('minutes', minutes);
+
+function getGreeting(World) {
+  return ('Hello World!');
+}
+
+var greeting = getGreeting('World');
+console.log('World', greeting);
+
+function addAndMultiplyBy5(x, y) {
+  return (x + y) * 5;
+}
+
+var result1 = addAndMultiplyBy5(10, 5);
+console.log('output #1', result1);
+
+function multiplyAndDivideBy5(x, y) {
+  return (x * y) / 5;
+}
+
+var result2 = multiplyAndDivideBy5(35, 10);
+console.log('output #2', result2);
+
+function subtractTwoNumbers(x, y) {
+  return x - y;
+}
+
+var result3 = subtractTwoNumbers(22, 7);
+console.log('result #3', result3);
+
+function getCircleCircumference(Radius) {
+  return (Radius);
+}
+
+var Radius = getCircleCircumference(31.41592653589793);
+console.log('Radius is', Radius);
+
+function getFullName(x, y) {
+  return (x + y);
+}
```

Comment:
Here the parameters should be `firstName` and `lastName`. The function should return a string with both the first and last names with a space between the names.